packages/engine/src/interaction/systems/InteractiveSystem.ts

```diff
@@ -96,7 +96,8 @@ export const onInteractableUpdate = (entity: Entity, xrui: ReturnType<typeof cre
       mat.opacity = opacity
     })
   })
-  ObjectFitFunctions.lookAtCameraFromPosition(xrui.container, transform.position)
+  const cameraTransform = getComponent(Engine.instance.cameraEntity, TransformComponent)
```

Comment:
what visual difference does this make?

Reply:
lookAtCameraFromPosition offset the hint and set its rotation to a value that didn't actually face the camera. Copying the camera quaternion yields the desired rotation and is the same pattern used for grabbable hints.

Reply:
when the user tilts their head over in XR, will this cause the XRUI to copy that rotation or will it be stuck to the XZ plane?

Reply:
Yes. This is also apparent with grabbables. Lots of copy/pasted code with these interactive xrui elements. I'll patch this for now but I still want to make a pr refactoring and simplifying this in the near future.